### PR TITLE
pymca-zocalo: new recipe

### PR DESCRIPTION
**Pull request by tschoonj**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pymca-zocalo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - pymca
+    - python
+    - setuptools
```
Comment (beckermr) [inline recipes/pymca-zocalo/meta.yaml]: Do you need setuptools at runtime?

File: recipes/pymca-zocalo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - xraylib
+
+test:
+  imports:
+    - pymca_zocalo
+
+about:
+  home: "https://github.com/DiamondLightSource/python-zocalo-pymca"
+  license: BSD
+  license_family: BSD
```
Comment (beckermr) [inline recipes/pymca-zocalo/meta.yaml]: Please specify which BSD license and use the proper SPDX identifier (e.g., `BSD-3-Clause`).

File: recipes/pymca-zocalo/meta.yaml (hunk 1 above)
Comment (tschoonj) [reply recipes/pymca-zocalo/meta.yaml]: yes

File: recipes/pymca-zocalo/meta.yaml (hunk 2 above)
Comment (tschoonj) [reply recipes/pymca-zocalo/meta.yaml]: Fixed

**Pull request merged**